pages/subscriptions.tsx

```diff
@@ -0,0 +1,117 @@
+import { ReactElement, useEffect, useState } from 'react';
+import Cookies from 'universal-cookie';
+import axios from 'axios';
+import { useRouter } from 'next/router';
+import { UserInfo } from '../components/types';
+import { gqlClient } from '../utils/courseAPIClient';
+import { PacmanLoader } from 'react-spinners';
+import { timeout } from 'q';
+
+const cookies = new Cookies();
+
+export default function SubscrptionsPage(): ReactElement {
+  const [userInfo, setUserInfo] = useState<UserInfo>(null);
```

Comment:
```suggestion
  const [userInfo, setUserInfo] = useState<UserInfo | null>(null);
```
We're trying to switch to strict null checking in the near future